### ISSUE TITLE
Feedback Request: Auto-accept transfers with Postboxes

### ISSUE THREAD

**Issue by betamos** [betamos/payload#7]
## TL;DR

I'm planning to add a feature called "Postboxes", which lets you auto-accept files from known devices.

**If you're using Payload, please share your thoughts on this idea.**

<img width="283" alt="Screen Shot 2022-11-11 at 6 22 14 PM" src="https://user-images.githubusercontent.com/135960/201452377-6ac491e5-864a-4aae-a71a-78620c930c44.png">

_Screenshot isn't final - take it with a grain of salt_

## Background

Currently, the receiver has to accept incoming transfers. Before you accept, you can optionally change the destination directory.  This is a good secure and flexible default. However, this means that all transfers requires interaction on both the sender and receiver devices.

There are two common cases where this can cause unnecessary friction:

1. You are frequently sending the same type of content, for instance your videos often goes from device A to device B, and always the same destination directory.
2. Your other computer is in a different room, so you have to move back and fourth between your devices.

## What is a postbox?

A postbox is created on the receiving side, and shared with the sender. Specifically:

- On device A, you create a postbox "Movies", by picking a name, icon and destination directory. You then share access to the postbox with devices B and C from your contact list. You only need to do this once.
- On devices B and C, you will be notified that A invited you to their postbox. When you choose who to send to, you can select devices directly (as today) or you can choose to send directly to a postbox.

Postbox transfers starts immediately. You can send multiple times to the same postbox.

## Privacy and security

- Only those you invite can see your postbox and transfer to it.
- Invitees can only see the name and icon of your postbox. The destination directory path and contents never leave the device.
- Invitees cannot see other invitees or their transfers.
- In the future, postboxes can be extended with limits such as max size, file type restrictions and automatic expiry.
